packages/fxa-shared/connected-services/models/AttachedClient.ts

```diff
@@ -31,7 +31,7 @@ export const attachedClientsDefaults: AttachedClient = {
   isCurrentSession: false,
   deviceType: null,
   name: null,
-  createdTime: null,
+  createdTime: Infinity,
```

Comment:
Was this done so that `createdTime` is always a number, instead of a `null | Number` before?